src/main/java/com/autotune/analyzer/recommendations/engine/RecommendationEngine.java

```diff
@@ -28,6 +28,7 @@
 import com.autotune.common.k8sObjects.K8sObject;
 import com.autotune.common.utils.CommonUtils;
 import com.autotune.database.service.ExperimentDBService;
+import com.autotune.metrics.KruizeNotificationCollectionRegistry;
```

Comment:
@msvinaykumar - Please update kruize documentation with the metrics being captured

Reply:
@msvinaykumar Can you update the documentation with the details, as discussed will need this if to update the kruize metrics script before running the scale test

Reply:
done